internal/schema/ize-spec.json

```diff
@@ -37,31 +46,40 @@
                 {
                     "type": "boolean"
                 }
-            ]
+            ],
+            "description": "plain text"
         },
         "prefer_runtime": {
-            "type": "string"
+            "type": "string",
+            "description": "prefer runtime"
         },
         "apps_path": {
-            "type": "string"
+            "type": "string",
+            "description": "path to apps directory"
         },
         "root_dir": {
-            "type": "string"
+            "type": "string",
+            "description": "project directory"
```

Comment:
```suggestion
            "description": "(optional) Project directory can be set here. By default it's the current directory, but in case you prefer to run ize from the outside of repo it may be useful (uncommon)."
```